src/lib/audit.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+import { db } from '@/database'
+import { AuditEventType, AuditRecordType } from '@/database/types'
+
+export const audit = async (
+    eventType: AuditEventType,
+    userId: string,
+    recordType: AuditRecordType,
+    recordId: string,
+): Promise<void> => {
+    console.log(`Auditing ${eventType}: ${recordType}/${recordId}`)
+    await db
+        .insertInto('audit')
+        .values({
+            userId: userId,
+            eventType: eventType,
+            recordType: recordType,
+            recordId: recordId,
+        })
+        .execute()
```

Comment:
I'd suggest `audit` function accept an object that could be passed as-is into `values`

Reply:
Thanks for the suggestions @nathanstitt! I addressed the review comments.